deploy: update resizer to v1.6.0 version

humblec opened the pull request:
This commit make deployment changes to consume v1.6.0 version of the resizer controller.

Reference:
https://github.com/kubernetes-csi/external-resizer/releases/tag/v1.6.0

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
deploy: update resizer to v1.6.0 version

This commit make deployment changes to consume v1.6.0
version of the resizer controller.

Reference:
https://github.com/kubernetes-csi/external-resizer/releases/tag/v1.6.0

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 reviewed (changes requested):
update in helm readme for both cephfs and rbd is missing

Madhu-1 commented:
@humblec, any reason to backport this to 3.7?

humblec commented:
> @humblec, any reason to backport this to 3.7?

its ok to not backport too. I have removed the label and also changed the readme. That readme has to be updated for other sidecars but I will put it as a seperate PR. @Madhu-1 ptal.. thanks
